bundles/org.openhab.core.model.yaml/src/main/java/org/openhab/core/model/yaml/internal/sitemaps/YamlSitemapDTO.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.model.yaml.internal.sitemaps;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.model.yaml.YamlElement;
+import org.openhab.core.model.yaml.YamlElementName;
+
+/**
+ * The {@link YamlSitemapDTO} is a data transfer object used to serialize a sitemap in a YAML configuration file.
+ *
+ * @author Mark Herwege - Initial contribution
+ */
+@YamlElementName("sitemap")
+public class YamlSitemapDTO implements YamlElement, Cloneable {
+
+    public String uid;
+    public String label;
+    public String icon;
+    public List<Map.Entry<String, YamlWidgetDTO>> widgets;
```

Comment:
We tried to use map but in this case we have no identifier for each widget so th elist seems to appropriate.